packages/TelegramClient-UI.package/TCUMessage.class/instance/messageModel..st

```diff
@@ -1,4 +1,4 @@
 accessing
-messageModel: aTCCMessage
+messageModel: aCoreMessage
```

Comment:
The parameters in the methods are called `aCoreMessage`, so why should the attribute have a different name and vice versa?

Reply:
There are two ideas at play here: On one hand are we trying to name every method attribute by their type, as there is no typesavety in Squeak this is the best way to ensure this while debugging.
On the other hand we want to make clear that this is part of a model-view-controller pattern.

Reply:
Ah okay 👍